src/stars.css

```
@@ -0,0 +1,44 @@
+.stars {
+  display: flex;
+  margin: 0;
+  padding-left: 0;
+  list-style-type: none;
+
+  /* justify-content: flex-start;
+  width: fit-content; */
+}
+
+.stars__star {
+  display: flex;
+  background: url(./images/star.svg) center no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 :nth-child(-n + 0) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
```

u can make it easier
```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
  background-image: url(images/star-active.svg);
}
```